### ISSUE TITLE
docs: dark mode

### ISSUE THREAD

**Issue by mikeshultz** [ApeWorX/ape#1924]
Would be super cool to have dark mode for the framework docs.  Painful to look at in the late hours.

**Comment by fubuloubu** [comment]
Yeah I want to explore moving to a better, non-custom theme with @dtdang down the road, maybe Shibuya:
https://vyper-tests.readthedocs.io/
